src/NLog/GlobalDiagnosticsContext.cs

```diff
@@ -111,5 +131,17 @@ public static void Clear()
                 dict.Clear();
             }
         }
+
+        internal static string RenderContextObject ( object o )
```

Comment:
Sure. I chose `RenderContextObject` because I assumed at some point in the future it was likely that objects would have custom renderers. But the name change will work in either context.